### PR TITLE
simple-reminder: add initial version of the extension

### PR DESCRIPTION
**Pull request by comoser**
## Description

Add new `Simple Reminder` extension.

This extension lets the user set reminders by using natural language (in plain english only) and without having any dependency on third party applications. When the time for the reminder is reached then the extension will notify the user with a system notification.

Full features:
- "Add-reminder" command to set a new reminder in the system by interpreting natural language
- List of the existing reminders
- Action to delete existing reminders
- Background task to notify the user with a system notification when the reminders date and time is up

## Screencast

simple-reminder-1
simple-reminder-2
simple-reminder-3
simple-reminder-3

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

**Comment by pernielsentikaer** [comment]
I'm not sure the "Set Reminder" list item is the best way, depending on how many items you have you could do something like it's done in this extension: https://www.raycast.com/maggie/todo-list

**Comment by comoser** [comment]
> I'm not sure the "Set Reminder" list item is the best way, depending on how many items you have you could do something like it's done in this extension: https://www.raycast.com/maggie/todo-list

I’ll take a look at it, and try to spot the location where I can actually listen to an Enter event. Ty for the suggestion

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥
